email_automation/email-config.py

```diff
@@ -0,0 +1,171 @@
+import boto3
+from botocore.exceptions import ClientError
```

Comment:
These modules are not in the e-mission environment, or in this repo.
The script will not run as-is.
You need to have setup instructions and/or a setup script.

Reply:
I've written a github actions workflow file (since we'll have to have one anyway) and included this section in the steps:
```
- name: Install Boto3
    run: pip install boto3
```
I'll commit it after we discuss AWS authentication in our meeting tomorow, since I will likely have to add a section to support the GitHub OIDC provider.